src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -455,6 +463,14 @@
                     :TokenID   token-id
                     :ChainID   to-chain-id))
 
+      (= bridge-name constants/bridge-name-erc-1155-transfer)
```

Comment:
I haven't reviewed PRs that evolved this `send.events` namespace, but the fact that it has zero coverage is not good. This namespace is not trivial and I'd normally reject as a reviewer. The re-frame architecture was built to help make this layer testable and it's the top-priority recommended by re-frame to test https://github.com/day8/re-frame/blob/5bd82b3d6625af1fac2b21ba0cd5bab448e44ffe/docs/Testing.md#what-to-test.

I know it's not a tech debt from your PR and at this stage it might be too much work to attach this debt. But, it's never too late to play the boy scout rule `Always leave your code cleaner than you found it`.

Perhaps something you guys could figure out in future changes to this `events` namespace.

Reply:
Thanks @ilmotta  I'll consider it for new PRs :)

cc: @status-im/wallet-mobile-devs